R/numbered_checks.R

```diff
@@ -372,4 +372,31 @@ check_condate_hdcdate_dthdate <- function(form){
              (deceased == 1 & parsed_condate > parse_dmY(dthdate)))) %>%
     mutate(parsed_condate = NULL)
   return(problems)
+}
+
+#' The time difference between the time from injury to hsp admission and 
+#' the time from injury to prep solution in ER is within +/- 24 hr range
+#' 
+#' @param form a dataframe containing form1.1, form4.1, form5.14
+#' @return a dataframe containing problematic entries
+#' 
+#' @import tidyverse
+#' @export
+check_ihunits_ipunits <- function(form){
+  problems <- form %>% mutate(
+    ihtime = if_else(ihunits == 1, ihhrs,
+                     if_else(ihunits == 2, ihdays * 24, as.numeric(NA))),
+    iprep1_time = if_else(ipunits1 == 1 & !is_invalid(ipreph1), ipreph1,
```

Comment:
Do no check for invalids in numbered checks, we will reserve them to the check_invalids.